### ISSUE TITLE
Run environment once a week

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-firmware#1307]
Using schedules. What is annoying is that we can't have schuduled and manual job at the same time AFAIK. Are we okay to ditch the manual option? We can use the same trick as we do with hw tests: run the job for branches prefixed with `env/` or similar.

**Comment by tsusanka** [comment]
Let's have two jobs one for manual, one for schedules.

**Comment by vdovhanych** [comment]
Not really. You can go ahead and run the scheduled job manually from the "Schedules" page in the project ci/cd settings. Shown [here](https://docs.gitlab.com/ee/ci/pipelines/schedules.html#running-manually).

**Comment by prusnak** [comment]
> Moved to [satoshilabs/devops#48](https://github.com/satoshilabs/devops/issues/48).

Should we close this issue, then?
